src/lib.rs

```diff
@@ -247,7 +244,7 @@ pub(crate) unsafe fn rav1d_open(c_out: &mut *mut Rav1dContext, s: &Rav1dSettings
         return Err(ENOMEM);
     }
     let stack_size: usize = 1024 * 1024 * get_stack_size_internal(&mut thread_attr);
-    pthread_attr_setstacksize(&mut thread_attr, stack_size);
+    pthread_attr_destroy(&mut thread_attr);
```

Comment:
Why destroy `thread_attr` here? Don't we already have logic for destroying it at the appropriate time?

Reply:
Nope, we deleted it accidentally I think.